src/Main/MainGui.cpp

```diff
@@ -189,9 +182,8 @@ int main( int argc, char ** argv )
     App::Application::Config()["SplashTextColor" ] = "#ffffff"; // white
     App::Application::Config()["SplashInfoColor" ] = "#c8c8c8"; // light grey
 
-#if (QT_VERSION >= QT_VERSION_CHECK(5, 7, 0))
     QGuiApplication::setDesktopFileName(QStringLiteral("org.freecadweb.FreeCAD.desktop"));
-#endif
+
 

```

Comment:
This will be three new lines in a row, maybe it could be reduced to just one?

Reply:
It was really only two, but I force-pushed it down to one :smile:.